breakwater/src/prometheus_exporter.rs

```diff
@@ -56,11 +56,11 @@ impl PrometheusExporter {
             statistics_information_rx,
             metric_legacy_ips: register_int_gauge(
                 "breakwater_ips",
```

Comment:
One problem is that it changes the semantics of the metrics.
We can either
1. silently break semantics (this PR)
2. keep the old metrics and add new ones, e.g. `breakwater_ips_v4` and `breakwater_ips_v6`
3. Only add the new ones -> We will break existing dashbaords

I personally would prefer 2, so that we don't break anything. Any opinions?

Reply:
For the record: We are now going with 3.